dhis-2/dhis-support/dhis-support-hibernate/src/main/java/org/hisp/dhis/hibernate/DefaultHibernateConfigurationProvider.java

```diff
@@ -240,6 +240,12 @@ private Properties getHibernateProperties()
             putIfExists( "false", "hibernate.cache.use_query_cache", props );
         }
 
+        // Enable Hibernate statistics if Hibernate Monitoring is enabled
+        if ( configurationProvider.getProperty( ConfigurationKey.MONITORING_HIBERNATE_ENABLED ).equals( "true" ) )

```

Comment:
Could we switch to use "on" and "off" as config values? That was the intent from the beginning, see e.g. key METADATA_AUDIT_PERSIST (I know Morten broke the convention with the AMQP_EMBEDDED_SECURITY etc).

Then, instead of doing equals( "true" ) you can do DhisConfigurationProvider.isEnabled( ConfigurationKey ).

Since we have not yet released the monitoring part it is okay to switch config syntax.

Reply:
ok